edx_proctoring/api.py

```diff
@@ -2003,13 +2002,9 @@ def get_student_view(user_id, course_id, content_id,
             return None
 
         # Just in case the due date has been changed because of the
-        # self-paced courses, use the due date from the context and
-        # update the local exam object if necessary.
+        # self-paced courses, use the due date from the context
+        # import pdb;pdb.set_trace()
```

Comment:
```suggestion
```

Reply:
> What's the migration plan for due dates into edx-when? 

The migration story for prod is in [DEVOPS-8619](https://openedx.atlassian.net/browse/DEVOPS-8619). For dev, there's no risk. Everything already falls back on the date in the modulestore. If someone is intending on developing date extensions, they'll have to republish the course (which happens if you change the due date in studio, anyway).

> 
> Also, wanted to flag that we'll end up in conflict between this and #559

I'll reversion this as 1.7.1